pytensor/tensor/slinalg.py

```diff
@@ -472,6 +475,7 @@ def solve_triangular(
             trans=trans,
             unit_diagonal=unit_diagonal,
             check_finite=check_finite,
+            overwrite_b=overwrite_b,
```

Comment:
This has to be done via inplace rewrites, and should just be ignored like we do in other OPs: https://github.com/pymc-devs/pytensor/blob/54a2c6e06b4dc7fb3b91340cf139994667120cfb/pytensor/tensor/slinalg.py#L170-L172

Reply:
Instead of saying it is ignored, we should say this is applied during rewrites when it is safe to destroy the input